graphql/schemabuilder/reflect.go

```diff
@@ -612,8 +612,8 @@ func (sb *schemaBuilder) buildStruct(typ reflect.Type) error {
 }
 
 var scalars = map[reflect.Type]string{
-	reflect.TypeOf(bool(false)): "bool",
-	reflect.TypeOf(int(0)):      "int",
+	reflect.TypeOf(bool(false)): "Boolean",
+	reflect.TypeOf(int(0)):      "Int",
```

Comment:
I agree with you in that I don't like introducing an inconsistency in the type naming scheme.
But I also think there is a value to natively supporting types like `String`, `Boolean`.
One other thing to note is that when we return an `int64` type value from thunder to a client, we lose the ability to express all possible integers of `int64`, beyond +/-`Number.MAX_SAFE_INTEGER`. 

For both of what I mentioned above, I don't think we need to worry about it now, but this may come up in the future and could potentially introduce a weird behavior if someone's dealing with a large integer.